src/js/storage.js

```diff
@@ -461,9 +461,11 @@ BadgerPen.prototype = {
     this._setupDomainAction(domain, "", "userAction");
 
     // if Privacy Badger never recorded tracking for this domain,
+    // and it's not a DNT-compliant domain,
     // remove the domain's entry from Privacy Badger's database
```

Comment:
Do we need to remove at all here? This removal idea was introduced in 82fd5d899e2048437dd2921e498ea3a9ec00fc79.